### PR TITLE
Update services.coffee

### PR DESCRIPTION
**Pull request by ycleptkellan**
Update language.

### CODE REVIEW

**Comment by norkans7** [comment]
yep

File: static/coffee/flows/services.coffee [hunk 1]
```diff
@@ -412,45 +412,45 @@ app.factory 'Flow', ['$rootScope', '$window', '$http', '$timeout', '$interval',
...
 
-        { type: 'wait_message', name:'Wait for Response', verbose_name: 'Wait for response', split:'message response', filter:[TEXT,SURVEY] },
+        { type: 'wait_message', name:'Wait for Response', verbose_name: 'Wait for Message', split:'message response', filter:[TEXT,SURVEY] },
 
         # survey media types
-        { type: 'wait_photo', name:'Wait for a photo', verbose_name: 'Wait for photo', filter:[SURVEY] },
-        { type: 'wait_audio', name:'Wait for an audio recording', verbose_name: 'Wait for audio', filter:[SURVEY] },
-        { type: 'wait_video', name:'Wait for a video', verbose_name: 'Wait for video', filter:[SURVEY] },
+        { type: 'wait_photo', name:'Wait for a photo', verbose_name: 'Wait for Photo', filter:[SURVEY] },
+        { type: 'wait_audio', name:'Wait for an audio recording', verbose_name: 'Wait for Audio', filter:[SURVEY] },
```
Comment (ericnewcomer) [inline static/coffee/flows/services.coffee]: why capitalize audio, video, and photo?

File: static/coffee/flows/services.coffee [hunk 2]
```diff
@@ -412,45 +412,45 @@ app.factory 'Flow', ['$rootScope', '$window', '$http', '$timeout', '$interval',
...
-        { type: 'wait_recording', name:'Get Recording', verbose_name: 'Wait for recording', filter:VOICE },
-        { type: 'wait_digit', name:'Get Menu Selection', verbose_name: 'Wait for menu selection', filter:VOICE },
-        { type: 'wait_digits', name:'Get Digits', verbose_name: 'Wait for multiple digits', split:'digits', filter:VOICE },
+        { type: 'wait_recording', name:'Get Recording', verbose_name: 'Wait for a Recording', filter:VOICE },
+        { type: 'wait_digit', name:'Get Menu Selection', verbose_name: 'Wait for Menu Selection', filter:VOICE },
+        { type: 'wait_digits', name:'Get Digits', verbose_name: 'Wait for Multiple Digits', split:'digits', filter:VOICE },
 
         # online flows
-        { type: 'webhook', name:'Call Webhook', verbose_name: 'Call webhook', split:'webhook response', filter:[TEXT,VOICE] },
+        { type: 'webhook', name:'Call Webhook', verbose_name: 'Make a request to a Webhook', split:'webhook response', filter:[TEXT,VOICE] },
```
Comment (ericnewcomer) [inline static/coffee/flows/services.coffee]: Confused why some are changed to title case and some are not.

Comment (ycleptkellan) [inline static/coffee/flows/services.coffee]: I figured TextIt-specific elements should be capitalized in Actions, and capitalized those in the RuleSet operands as well for consistency.

File: static/coffee/flows/services.coffee (hunk 1 above)
Comment (ycleptkellan) [inline static/coffee/flows/services.coffee]: for consistency w/ the other changes, though certainly not a big deal.

File: static/coffee/flows/services.coffee (hunk 2 above)
Comment (ericnewcomer) [inline static/coffee/flows/services.coffee]: Right, but "Wait for Multiple Digits" is title case and "Make a request to a Webhook" is not.

Comment (ycleptkellan) [inline static/coffee/flows/services.coffee]: Had initially thought 'Make a request to a Webhook' should follow 'Send a Message to a different Contact' and 'Start a different Contact in a Flow'. Capitalize all nouns?

Comment (ericnewcomer) [inline static/coffee/flows/services.coffee]: Sprechen sie Deutsch? ;)

I think my preference is normal case, but I'm okay with treating our formal object types as proper nouns (Flow, Contact, Campaign, Message, Webhook, etc)

Comment (ycleptkellan) [inline static/coffee/flows/services.coffee]: Ich wünsche!

**Pull request closed without merging**